app/views/layouts/application.html.erb

```diff
@@ -40,6 +40,19 @@
     <%= content_for?(:body) ? yield(:body) : yield %>
     <%= render 'shared/ajax_modal' %>
     <%= javascript_include_tag 'application' %>
+    <link href="https://cdn.jsdelivr.net/npm/select2@4.0.13/dist/css/select2.min.css" rel="stylesheet" />
+
+    <%= javascript_tag do %>
+      oldDefine = define;
+      define = null;
+    <% end %>
+
+    <script src="https://cdn.jsdelivr.net/npm/select2@4.0.13/dist/js/select2.js"></script>
```

Comment:
Trying to install this with `yarn add` produced a number of conflicts in my `yarn.lock`. Additionally, I'm not certain that I would be able to avoid the `almond-rails` conflicts referenced above if I used a separate approach.